cloud_build/verify-provenance.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
```

Comment:
Can we add some notes about the expected exit codes of this script? e.g. this script will exit with a non zero exit code if the policy validation fails,